### PR TITLE
Fixes #545 - Addition of Auto-Assign Bot

### LINKED ISSUE #545: Addition of Assign-bot
🐞 **What bugs you?**
We should add an assign bot to automate assigning the issues/PR to willing contributors

🎯 **Goal**
1. Keep track of assigness
2. Increases the organization

💡 **Possible solutions**
1. Addition of assign-bot

📋  **Steps to solve the problem**

 1. It could also happen that someone claims but doesn't work. Need to take care of it for future contributors.
 2.  A tracking mechanism should be set up?

### PR DESCRIPTION
**Pull request by JeS24**
Fixes #545 

@shreyasbapat

### CODE REVIEW

**Comment by shreyasbapat** [comment]
Looks good. I'll need to install some app?

**Comment by JeS24** [comment]
> Looks good. I'll need to install some app?

Yeah, this: https://probot.github.io/apps/auto-assign/

**Comment by JeS24** [comment]
I have requested an install for EinsteinPy: @shreyasbapat 
image

File: .github/auto_assign.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Set to true to add reviewers to pull requests
+addReviewers: true
+
+# Set to true to add assignees to pull requests
+addAssignees: false
+
+# A list of reviewers to be added to pull requests (GitHub user name)
+reviewers:
+  - shreyasbapat
+  - ritzvik
```
Comment (shreyasbapat) [inline .github/auto_assign.yml:10]:
```suggestion

```

**Comment by JeS24** [comment]
@shreyasbapat Have you configured the bot or should I? I was thinking to give it access to just this repo.

**Comment by JeS24** [comment]
@shreyasbapat Oh, actually, you have to configure it. I can't.

**Comment by shreyasbapat** [comment]
> @shreyasbapat Oh, actually, you have to configure it. I can't.

Okay. I think I did not configure it on the repo.

**Comment by shreyasbapat** [comment]
Let's see how that works :P

**Pull request merged**